app/src/server/tasks/evaluateTestSetEntry.task.ts

```diff
@@ -60,7 +71,7 @@ export const evaluateTestSetEntry = defineTask<EvaluateTestSetEntryJob>({
     }
 
     const cacheKey = hashObject({
-      fineTuneId,
+      modelId,
       input: prunedMessages,
```

Comment:
Unrelated to this PR but we need to add the function_call and functions to this hash as well.

Reply:
Added